scripts/aio_cleanup.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python
+import datetime
+import dateutil.parser
+import os
+
+from novaclient import client
+
+SLAVE_NAME = os.environ.get('SLAVE_NAME', 'Jenkins-OM-AIO')
+NUM_HRS = os.environ.get('NUM_HRS', 6)
```

Comment:
I tried setting NUM_HRS as an environment variable, and it looks like os.environ.get() returns a String when the environment is set because I later got a TypeError.

```
$ export NUM_HRS=168
$ python aio_cleanup.py

Traceback (most recent call last):
  File "test.py", line 37, in <module>
    main()
  File "test.py", line 26, in main
    num_hrs_ago = now_local_tz - datetime.timedelta(hours=NUM_HRS)
TypeError: unsupported type for timedelta hours component: str

```